### PR TITLE
zg and rg distributions for each hard splitting in the history

### PR DESCRIPTION
**Pull request by klapidus**
A couple of new histograms in the user task

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDrop.cxx [hunk 1]
```diff
@@ -403,11 +407,19 @@ void AliAnalysisTaskSoftDrop::SoftDropDeepDeclustering(fastjet::PseudoJet jet, c
     Float_t pt1 = jet1.pt();
     Float_t pt2 = jet2.pt();
 
+    Float_t dr = sqrt( jet1.plain_distance(jet2) );
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDrop.cxx]: I think sqrt will needs an explicit namespace (?)

Comment (klapidus) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDrop.cxx]: I think it's ok as is (was already used in another method). Nevertheless, changed. Thanks.

Comment (raymondEhlers) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDrop.cxx]: Apologies - I really thought this was going to be an issue on root 6 - I guess I should have tested it! Thanks for bearing with me!

**Comment by raymondEhlers** [comment]
+1

**Pull request merged**